horace_core/sqw/file_io/@faccess_sqw_v4/update_sqw.m

```diff
@@ -0,0 +1,3 @@
+function obj = update_sqw(obj,mod_sqw,varargin)
+%UPDATE_SQW given initalized sqw file accessor, store modified sqw
+% information within sqw file.
```

Comment:
Guessing you can't update v4 files because it's the latest version, and that other versions will have a larger overload. Please add some comment so we know this (or the real reason if my guess is wrong) and can add updating functionality if/when it becomes possible

Reply:
This is actually part of other ticket (#1320) which have not been implemented yet. The name is indeed not very suitable. 

Previous versions were uncapable of this, and v4 routinely allows field modifications.  But no appropriate method to reflect this.